deeplearning4j-nlp-parent/deeplearning4j-nlp/src/main/java/org/deeplearning4j/text/tokenization/tokenizer/NGramTokenizer.java

```diff
@@ -47,7 +47,7 @@ public NGramTokenizer(Tokenizer tokenizer,Integer minN, Integer maxN) {
             for (int i = minN; i < min; i++) {
                 for (int j = 0; j < nOriginalTokens - i + 1; j++) {
                     List<String> originalTokensSlice = this.originalTokens.subList(j, j+ i);
-                    this.tokens.add(StringUtils.join(" ", originalTokensSlice));
+                    this.tokens.add(StringUtils.join(originalTokensSlice, " "));
```

Comment:
I think the method params just got mixed up

The method signature for .join is
```
    public static String join(Iterable<?> iterable, String separator) {
        return iterable == null?null:join(iterable.iterator(), separator);
    }
```

the collection comes first, the character to join with comes second.

given a collection 

```java
originalTokensSlice = ["foo", "bar", "baz"]
```

`StringUtils.join(" ", originalTokensSlice)` will return `" [foo, bar, baz]"`

where as 

`StringUtils.join(originalTokensSlice, " " )` will return `"foo bar baz"`

which is i think what we want, e.g. https://github.com/deeplearning4j/deeplearning4j/pull/2820/files#diff-9bca4ca3024bc576e4171421119da2ebR56